new_tutorials/04_distilling_a_reader.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Distil a Reader\n",
+    "\n",
+    "- **Level**: Advanced\n",
+    "- **Time to complete**: 45 minutes\n",
```

Comment:
Hmm these times are really long. Maybe we can divide them into the time it takes to actually do something and the time you have to wait for the model to process sth?